lib/generate-report.js

```diff
@@ -367,11 +371,16 @@ function generateReport(options) {
     function _createFeaturesOverviewIndexPage(suite) {
         const featuresOverviewIndex = path.resolve(reportPath, INDEX_HTML);
 
+        // Assign the correct overview template
+        var featuresOverviewTemplate;
```

Comment:
Please write this as and try to avoid the usage of `var`

`FEATURE_METADATA_OVERVIEW_TEMPLATE = suite.customMetadata ? FEATURE_CUSTOM_METADATA_OVERVIEW_TEMPLATE : FEATURE_METADATA_OVERVIEW_TEMPLATE;`